sc-cli.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
can you move this into the "cmd" dir since its local to that stuff?
You should probably also copy the Makefile from the k8s/service_controller to build an exe